### PR TITLE
Extract all translatable strings and add Simplified Chinese translation.[chore]

### PR DESCRIPTION
**Pull request by dongfengweixiao**
issue:
  1. src\components\Modals\ShareLimit\ShareLimitInput.vue
     :label="`${title} Limit`"
  2. src\components\Modals\TagsAndCategories\CreateCategoryDialog.vue
     all tips not work
  3. src\components\Modals\TorrentDetailModal\Tabs\Trackers.vue
     no idea about formatTrackerStatus

QQ截图20220428115840
QQ截图20220428115904
QQ截图20220428115916
QQ截图20220428115926
QQ截图20220428115938
QQ截图20220428115950
QQ截图20220428120001

# PR Checklist
- [x] I've started from master
- [x] I've only committed changes related to this PR
- [x] All Unit tests pass
- [x] I've removed all commented code
- [x] I've removed all unneeded console.log statements

### CODE REVIEW

**Comment by WDaan** [comment]
Great! This is fantastic! Thank you :)

**Comment by dongfengweixiao** [comment]
> Great! This is fantastic! Thank you :)

hi，how to extract string like ``:label="`${title} Limit`"``?

File: src/components/Modals/TagsAndCategories/CreateCategoryDialog.vue [hunk 1]
```diff
@@ -69,14 +69,14 @@ export default {
   },
   data: () => ({
     nameRules: [
-      v => !!v || 'Category name is required',
+      v => !!v || this.$i18n.t('modals.newCategory.tipOnNoName'),
```
Comment (WDaan) [inline src/components/Modals/TagsAndCategories/CreateCategoryDialog.vue:72]: This doesn't seem to work 😞

Comment (WDaan) [reply src/components/Modals/TagsAndCategories/CreateCategoryDialog.vue:72]: found a way to fix it though

**Comment by WDaan** [comment]
Thank you for putting in all the work and providing translations!! 😄

File: src/components/Modals/TagsAndCategories/CreateCategoryDialog.vue (hunk 1 above)
Comment (dongfengweixiao) [reply src/components/Modals/TagsAndCategories/CreateCategoryDialog.vue:72]: > found a way to fix it though

I don't have Vue relevant skills. The frontend is too difficult!

Comment (WDaan) [reply src/components/Modals/TagsAndCategories/CreateCategoryDialog.vue:72]: No worries! It has become quite complex in some areas, I don't blame you 😅

**Pull request merged**